src/js/common/schemaform/save-load-actions.js

```diff
@@ -350,18 +364,30 @@ export function removeInProgressForm(formId, migrations) {
 
       return Promise.resolve(res);
     }).then((res) => {
-      // If there’s some error when deleting, there’s not much we can
-      // do aside from not stop the user from continuing on
-      if (!res || res.status !== 401) {
-        window.dataLayer.push({
-          event: `${trackingPrefix}sip-form-start-over`
-        });
-        // after deleting, go fetch prefill info if they’ve got it
-        return dispatch(fetchInProgressForm(formId, migrations, true));
+
+      if (startOver) {
```

Comment:
I wonder if a better approach would be to make another action in the profile application and not add the branching logic in here. I think this is somewhat confusing to read and there's not a whole lot of common code, outside of the initial `fetch` call. If we needed to, we could make a function for wrapping the initial fetch call and use it both here and in the action in the profile app.

Reply:
That's a better idea, thanks @jbalboni.